pom.xml

```diff
@@ -47,7 +47,7 @@
         <module>empty-plugin</module>
         <module>hello-world</module>
         <module>global-configuration</module>
-        <!-- if adding new archetypes please also update .github/dependabot.yml -->
+        <!-- if adding new archetypes please also update .github/renovate.json -->
```

Comment:
is this comment still needed? won't they just be auto-detected?

Reply:
> won't they just be auto-detected?

Absolutely, this was introduced when refactoring the file through IJ.